src/components/Modal/Modal.tsx

```diff
@@ -171,7 +171,9 @@ export const ModalContent = (props: ModalContentProps) => {
  * NOTE: You must have at least one focusable element in the modal contents, for keyboard
  * accessibility reasons. (The close button counts as a focusable element.)
  *
- * @example
```

Comment:
Because of this line, nothing below this point was showing up.

<img width="1360" alt="" src="https://user-images.githubusercontent.com/7761701/200638716-019079ca-f68b-4933-9eb3-664eab08e032.png">

Reply:
Weird. What kind of docs should we revise for avoiding these kinds of jsdoc tags?

Reply:
Sorry, I'm not following. Do you mean "advise"? I was planning on just putting `@deprecated` at the end of the comments and avoiding using the others until this gets fixed.

Reply:
Sorry to clarify, it seems that Storybook isn't happy with certain jsdoc tags like `@example`. Do we have a place to put that information so we don't add them in the future?

Reply:
Oh I see. I can can add it to the code guidelines. Good call 👍